### DISCUSSION TITLE
Convert localy imported images to base64

### DISCUSSION THREAD

**Discussion by Tomekmularczyk** [resend/react-email#870] [Ideas]
### Goals

1. Store images locally
2. Import them in email template file 
3. Have them automatically converted to base64

### Background

Currently if you want to use images for backgrounds or icons you need to host them somewhere. Many projects support importing images just like any other files and handles transformations for them. By keeping them locally it's just much easier to manage them and by converting them to base64 its gonna improve the performance plus resolve problems with caching.

### Proposal

not sure

**Comment by Tomekmularczyk** [comment]
after spending some time investigating it, a lot of mail clients won't display base64 encoded images, for example GMAIL.

**Reply by cotwitch** [reply]
@Tomekmularczyk (Despite the fact that I know that base64 encoding is not very compatible among email clients) solely for testing purposes, I'm trying to import a JPG file and possibly encode it as base64 string/dataURI and use it via `<img src={importedImage}`.
Unfortunately, doesn't matter if I use `import` or `require`, result is the same - I'm getting error: 
`ERROR: No loader is configured for ".jpg" files`. 

So, I've tried to update `next.config.js` in a manner to webpack use "url-loader" for JPG files, but it's failing (it breaks Next's.js native/default way to handle/import images).

So, I would like to kindly ask you if you personally get past importing images in base64 data URI format? :))) Or have you give-up prior to this point? :)))
